cmd/thanos/main.go

```diff
@@ -29,10 +28,6 @@ import (
 )
 
 func main() {
-	// We use mmaped resources in most of the components so hardcode PanicOnFault to true. This allows us to recover (if we can e.g if queries
```

Comment:
Why do we remove this? Is this intentional?

cc @yashrsharma44

Reply:
this seems like it was left-over from testing. Should be reverted